src/java/grl/src/grl/driver/GRL_Driver.java

```diff
@@ -3,36 +3,40 @@
 import static com.kuka.roboticsAPI.motionModel.BasicMotions.positionHold;
 import static com.kuka.roboticsAPI.motionModel.BasicMotions.ptp;
 
-import grl.FRIMode;
 import grl.ProcessDataManager;
 import grl.StartStopSwitchUI;
 import grl.TeachMode;
 import grl.UpdateConfiguration;
 import grl.ZMQManager;
+import grl.UDPManager;
 import grl.flatbuffer.ArmState;
-import grl.flatbuffer.CartesianImpedenceControlMode;
 import grl.flatbuffer.KUKAiiwaInterface;
+import grl.flatbuffer.KUKAiiwaMonitorState;
+import grl.flatbuffer.KUKAiiwaState;
+import grl.flatbuffer.KUKAiiwaStates;
 import grl.flatbuffer.MoveArmJointServo;
 import grl.flatbuffer.MoveArmTrajectory;
+import grl.flatbuffer.Wrench;
 
+import java.io.IOException;
+import java.net.UnknownHostException;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.TimeoutException;
+import java.util.concurrent.locks.Lock;
+import java.util.concurrent.locks.ReentrantLock;
 
+import com.google.flatbuffers.FlatBufferBuilder;
 import com.google.flatbuffers.Table;
-
-// Compiler error here? COMMENT ALL FLEXFELLOW AND MEDIAFLANGEIOGROUP lines
```

Comment:
I can't make sense of this flexfellow, would you please fix the parts that I might have deleted?